src/tools/export_file_tool.rb

```diff
@@ -25,8 +25,9 @@ def activate
   end
 
   def export_with_file_dialog(triangle = nil)
-    @export_path = Configuration::JSON_PATH if @export_path.nil?
+    @export_path = Configuration::JSON_PATH if @last_export_path.nil?
     @export_path = UI.savepanel('Export JSON', @export_path, 'export.json')
+    @last_export_path = File.dirname(@export_path) unless @export_path.nil?
```

Comment:
where do you actually use `@last_export_path` when it's set? Shouldn't you set `@export_path=@last_export_path` or something similar?

Reply:
Ah, I think I missed to change it in line 29. So that should be
```@export_path = UI.savepanel('Export JSON', @last_export_path, 'export.json')```,
so we open the last opened path again.